### ISSUE TITLE
Prevent thrown Error for an active poll

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#3807]
**Current behaviour:**
If any poll is running and you navigate to the autopilot, then the following Errors (or similar ones) are thrown in the console:
```
data-store.service.ts:314 ERROR 
TypeError: Cannot read properties of undefined (reading 'includes')
    at AssignmentPollVoteComponent.defineVoteOptions (base-poll-vote.component.ts:489:42)
    at AssignmentPollVoteComponent.ngOnInit (base-poll-vote.component.ts:176:14)
    at callHookInternal (core.mjs:4024:14)
    at callHook (core.mjs:4051:13)
    at callHooks (core.mjs:4006:17)
    at executeInitAndCheckHooks (core.mjs:3956:9)
    at refreshView (core.mjs:13513:21)
    at detectChangesInView (core.mjs:13663:9)
    [...]
```

and other example:
```
TypeError: Cannot read properties of undefined (reading 'length')
    at AssignmentPollVoteComponent_ng_template_1_ng_container_1_ng_container_1_div_3_div_2_b_2_Template (base-poll-vote.component.html:67:29)
    at executeTemplate (core.mjs:12003:13)
    at refreshView (core.mjs:13498:13)
    at detectChangesInView (core.mjs:13663:9)
    at detectChangesInEmbeddedViews (core.mjs:13606:13)
    at refreshView (core.mjs:13522:9)
    at detectChangesInView (core.mjs:13663:9)
    at detectChangesInEmbeddedViews (core.mjs:13606:13)
    at refreshView (core.mjs:13522:9)
    [...]
```

**Reproduction:**
1. create any poll (assignment, topic or motion)
2. start the poll
3. navigate to the autopilot 
4. open the console
5. reload page
6. Errors are thrown

**Wanted behaviour:**
the Errors in the console should be prevented

**Comment by reiterl** [comment]
Mmh, I followed the steps. I tried a superuser and a meeting account. But I don't see the described errors or similar errors.

**Comment by bastianjoel** [comment]
This is already fixed with #3796
